cmd/snap-confine/snappy-app-dev

```diff
@@ -15,7 +15,7 @@ MAJMIN="$4"
 [ -n "$DEVPATH" ] || { echo "no devpath given" >&2; exit 1; }
 [ -n "$MAJMIN" ] || { echo "no major/minor given" >&2; exit 0; }
 
-APPNAME=`echo $APPNAME | tr '_' '.'`
```

Comment:
Oh, this script slows down execution of snap-confine. We should re-factor this a little!